index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div id="main">
+      <h1 id="title" style="text-align: center">John Pombe Joseph Magufuli</h1>
+      <div id="img-div" style="margin: auto; width: 95%">
+        <img
+          src="https://ichef.bbci.co.uk/news/976/cpsprodpb/57D6/production/_109368422_gettyimages-494936334.jpg"
+          alt="pombe magufuli"
+          id="image"
+          style="
+            height: auto;
+            object-fit: contain;
+            object-position: center;
+            display: block;
+            margin: auto;
+          "
+        />
+        <p
+          id="img-caption"
+          style="text-align: center; font-weight: bold; font-size: large"
+        >
+          5th President of Tanzania
+        </p>
+      </div>
+      <div
+        id="tribute-info"
+        style="text-align: center; width: 80%; padding: 5px 25px; margin: auto"
```

Comment:
- [x] [REQUIRED] Good job so far! This [review suggestion](https://github.com/Agetuni/CodeReview_Microverse/pull/1/files#r913107169) also like `[OPTIONAL]` because it is not **required changes**.

Reply:
[OPTIONAL] it would be great if you don't use inline styling inside the element tag.